lib/vm/src/trap/traphandlers.rs

```diff
@@ -524,6 +524,7 @@ impl Trap {
 ///
 /// Wildly unsafe because it calls raw function pointers and reads/writes raw
 /// function pointers.
+#[inline(always)]
```

Comment:
Seeing lots of `inline(always)` so I'll say what I remember about them:

`#[inline]`: consider this function for cross-crate inlining even without LTO (has no effect if LTO is enabled. has no effect on code within the same crate [citation needed (this may be true if compiling with high optimization settings but not in general)])

`#[inline(always)]`: inline this function in all contexts even if the optimizer thinks it'll make things slower.

`#[inline(always)]`  is a pretty sharp tool because Rust does inlining optimization by default if it thinks it'll make things faster. It can be useful when you really need it, but you usually don't.

https://nnethercote.github.io/perf-book/inlining.html for more info about this